### ISSUE TITLE
Not validating properly (IPV4 with block >= 256 still returns true)

### ISSUE THREAD

**Issue by nolawnchairs** [indutny/node-ip#105]
Hi,

The methods `ip.isV4Format()` and `ip.isV6Format()` are returning `true` for an invalid IP address:

```javascript
const ipAddress = "103.23.133.256";
console.log(ipAddress, ip.isV4Format(ipAddress), ip.isV6Format(ipAddress));
// > 103.23.133.256 true true
```
Should this not validate each block so that V4 blocks > 255 would case the function to return `false`? Also, since V4 and V6 addresses have different delimiters (colons/periods) that passing a V4 address into the `isV6Format` function should return false as well?

Would it be possible to add a validation method that accepts both V4 and V6 addresses and validates them?

**Comment by slavikme** [comment]
You are right, this should return `false`. Seems like a buggy implementation.

Anyway, you could verify it using this method:
```js
function isValidIPv4 (address) {
  return ip.isV4Format(address) && ip.cidr(address+'/32') === address
}
```
